modules/react/button/stories/visual-testing/utils.tsx

```diff
@@ -31,6 +31,7 @@ export const stateTableColumnProps = [
   {label: 'Focus Hover ', props: {className: 'focus hover', disabled: false}},
   {label: 'Active ', props: {className: 'active', disabled: false}},
   {label: 'Active Hover ', props: {className: 'active hover', disabled: false}},
+  {label: 'Mirrored Icon ', props: {shouldMirrorIcon: true, disabled: false}},
```

Comment:
This was the simplest way I could think to add a visual test for mirroring these icons, but I'm open to other suggestions as well.